src/Cli/Commands/InitOptions.cs

```diff
@@ -105,8 +105,8 @@ public class InitOptions : Options
         [Option("graphql.disabled", Default = false, Required = false, HelpText = "Disables GraphQL endpoint for all entities.")]
         public bool GraphQLDisabled { get; }
 
-        [Option("rest.request-body-strict", Default = CliBool.True, Required = false, HelpText = "When set to true, allows extraneous fields in the request body for REST.")]
-        public CliBool RestRequestBodyStrict { get; }
+        [Option("rest.request-body-strict", Required = false, HelpText = "When set to true, allows extraneous fields in the request body for REST.")]
```

Comment:
So by excluding `Default = CliBool.True`, you require the user to include a value? Can you add that as a comment?